### PR TITLE
Web copy for the badge flair contest

### PR DESCRIPTION
**Pull request by thatdocslady**
This PR is the base draft content for the badge flair contest. I don't know how to really work the insides of the website and it seems that the FAQ page is the only reasonable location for the text, so I used the existing format hopefully correctly.

WIP until the following happens:

- [x] Verify the location of the decorating station
- [x] Validate the new section and build the website locally (or pre-merge CI if it exists?)

Main reviewer is @bexelbie and he will help with the TODO items before we merge.

### CODE REVIEW

File: templates/www/cz/2019/devconfcz-2019-faq.html [hunk 1]
```diff
@@ -186,6 +187,45 @@ <h2 id="speaker">{{ _('Speaker Questions') }}</h2>
...
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: What are the decoration guidelines? Are there any restrictions?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('We want you to be creative and have fun, but we also want to make sure the contest is fair and inclusive. Please keep in mind the following guidelines:') }}</p>
+        <ul>
+          <li>{{ _('You should not cover any part of your name, but you can cover the DevConf logo or other elements if you don't need to acces them.') }}</li>
+          <li>{{ _('All contest entires must follow the Code of Conduct. Do not use any harmful messaging or images in your decorations.') }}</li>
+          <li>{{ _('Although you may use your own items for your badge (such as markers or stickers), we value having similar resources for everyone, so if you overdo it with items you brought yourself (such as LED lights or sparkly feathers), you may be sadly disqualified.') }}</li>
```
Comment (bexelbie) [inline templates/www/cz/2019/devconfcz-2019-faq.html]: sadly is odd here, can we delete it or rephrase?  It just reads weird to me.

File: templates/www/cz/2019/devconfcz-2019-faq.html [hunk 2]
```diff
@@ -186,6 +187,45 @@ <h2 id="speaker">{{ _('Speaker Questions') }}</h2>
...
+        <a href="#" class="toggle-title"><strong>{{ _('Q: What is this badge flair contest anyway?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('The concept of the badge flair contest started at <a href="http://www.writethedocs.org/conf/prague/2018/badge-flair/">Write the Docs</a>, a conference and community for documentation-oriented tech folks who love to start fun and light-hearted traditions. The idea is simple: Decorate your badge, tweet a picture, and you might win a prize!') }}</p>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: Who can participate? Do I have to participate?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('All conference attendees and volunteers are eligible to participate. Organizing team members cannot enter the contest, although we do encourage you to lead by example and spread the word.') }}</p>
+          <p>{{ _('<u>The contest is optional.</u> Of course, you are free to use the decorating station even if you don’t want to participate, and decorate your badge for your own enjoyment!') }}</p>
```
Comment (jridky) [inline templates/www/cz/2019/devconfcz-2019-faq.html]: Don't use <u> tag. It's better to use <strong> / <b> tags.

File: templates/www/cz/2019/devconfcz-2019-faq.html [hunk 3]
```diff
@@ -186,6 +187,45 @@ <h2 id="speaker">{{ _('Speaker Questions') }}</h2>
...
+            <li>{{ _('<u>The winner must be in the room at the time of the announcement.</u> If they aren’t present, the prize will go to the runner-up. Follow the <a href="https://twitter.com/devconf_cz">DevConf CZ</a> Twitter account to stay updated with room announcements and other contest updates.') }}</li>
+          </ul>
+	</div>
+	</div>
+
+	<div class="toggle">
+        <a href="#" class="toggle-title"><strong>{{ _('Q: What are the decoration guidelines? Are there any restrictions?') }}</strong><i class="fa fa-plus"></i></a>
+        <div class="toggle-content"><p>{{ _('We want you to be creative and have fun, but we also want to make sure the contest is fair and inclusive. Please keep in mind the following guidelines:') }}</p>
+        <ul>
+          <li>{{ _('You should not cover any part of your name, but you can cover the DevConf logo or other elements if you don't need to acces them.') }}</li>
```
Comment (jridky) [inline templates/www/cz/2019/devconfcz-2019-faq.html]: It's prohibited to use  '  in the text inside of {{ _(' ') }}, replace it by ’ or use non-shorted version of the words (e.g. don't -> do not)

Comment (jridky) [reply templates/www/cz/2019/devconfcz-2019-faq.html]: for proper ’ see line 202 and word "don’t"

File: templates/www/cz/2019/devconfcz-2019-faq.html (hunk 2 above)
Comment (thatdocslady) [reply templates/www/cz/2019/devconfcz-2019-faq.html]: @jridky what do you mean by "/ tags"? I'm not familiar with this term, if you can give me exact syntax I'll happily replace it. Thanks!

**Comment by thatdocslady** [comment]
@bexelbie @jridky I implemented your feedback, could someone please do a quick local build to check if the page syntax still works?

Also, I'm not very good at squashing commits, so if ths PR is ready to merge your help is greatly appreciated!

**Comment by jridky** [comment]
all is fine. Merging.

**Pull request merged**